src/web/gwc/src/main/java/org/geoserver/gwc/web/GWCServiceDescriptionProvider.java

```diff
@@ -151,7 +154,7 @@ public List<ServiceLinkDescription> getServiceLinks(
             if (gwcConfig.isWMSCEnabled() && null != app.getBean("gwcServiceWMS")) {
                 links.add(
                         new ServiceLinkDescription(
-                                "wmts",
+                                SERVICE,
```

Comment:
The other changes are making sense, this one seems surprising... why go from a fixed "wmts" to a fixed "gwc?

Reply:
This is only to establish the headings; since you had me rename WMTSServiceDescriptorProvider to GWCServiceDescriptorProvider I was going with the flow.

Really the variable names have shifted over time to accommodate GWC speaking widely different protocols and planning ahead for ogcapi services. The variable protocol in the case of gwc provides "WMS-C", "WMTS", "TMS" etc...

Reply:
Updated the PR description to explain the relationships better